build/Microsoft.IIS.Administration.props

```diff
@@ -7,6 +7,8 @@
     <PublishRoot>$(SolutionDir)dist</PublishRoot>
     <AppPublishRoot>$(SolutionDir)dist\Microsoft.IIS.Administration</AppPublishRoot>
     <PluginsPublishRoot>$(SolutionDir)dist\Microsoft.IIS.Administration\plugins</PluginsPublishRoot>
+    <SignAssembly>true</SignAssembly>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
```

Comment:
Why is the condition " "$(PublishDir) == '' " removed for CopyLocalLockFileAssemblies?

Reply:
It's a long story:

We used to have a "packager" project that includes all plugins as assembly and that project would evaluate CopyLocalLockFileAssemblies to true in publish. Effective we were still doing CopyLocalLockFileAssemblies. That statement were there just to prevent the redundant operations.

That packager was causing a bunch of weird runtime behavior and essentially was just there the facilitate the .\publish.ps1 script I mentioned. The whole process is really wonky and now that I have removed packager and have each plugin build on its own, we no longer have that condition.

In short: We were effectively always doing it

Reply:
The condition `<PropertyGroup Condition="$(PublishDir) == ''">` has no effect, `CopyLocalLockFileAssemblies` is true on publish anyway...

Reply:
Then how about dotnet build - is CopyLocalLockFileAssemblies always true as well?

Reply:
From our existing `plugins.props` we would always set `CopyLocalLockFileAssemblies` on true. This line basically continues the same behavior